### PR TITLE
Update deprecated 'create-react-class' dependencies in tests to ES6 classes

### LINKED ISSUE #461: Update tests to not use create-react-class
Discovered that as part of https://github.com/chenglou/react-motion/pull/460

Caching `node_modules`  did us a bad service

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Fixes #461

CC: @nkbt

### CODE REVIEW

**Comment by nkbt** [comment]
Awesome!

**Comment by MichaelDeBoey** [comment]
@nkbt just have to fix the linting errors now :P

**Comment by MichaelDeBoey** [comment]
@nkbt Fixed :-)

File: bower.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "build/react-motion.map"
   ],
   "dependencies": {
-    "react": ">=0.13.2 || ^0.14 || ^15.0.0"
+    "react": "^0.14.9 || ^15.3.0"
```
Comment (nkbt) [inline bower.json]: oh, thanks for that

Comment (MichaelDeBoey) [inline bower.json]: You're welcome 😃

File: test/Motion-test.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
+/* eslint-disable class-methods-use-this */
```
Comment (nkbt) [inline test/Motion-test.js]: "fixed" =)

Apparently we can make those methods static according to http://eslint.org/docs/rules/class-methods-use-this

But that is OK for the moment. I'd rather get tests working now.

Comment (MichaelDeBoey) [inline test/Motion-test.js]: Yes indeed, but render isn't supposed to be static so that's why I did it like this

**Pull request merged**